src/main/java/uk/gov/companieshouse/accounts/association/service/AssociationsService.java

```diff
@@ -46,6 +55,9 @@ public class AssociationsService {
 
     private static final Logger LOG = LoggerFactory.getLogger(StaticPropertyUtil.APPLICATION_NAMESPACE);
 
+    public static final int DAYS_SINCE_INVITE_TILL_EXPIRES = 7;
```

Comment:
if used at 2 places, move it to application.properties so that we can change it in the properties.

i would say either duplicate it in each class or move it to statipropertyutil

Reply:
https://github.com/companieshouse/accounts-association-api/pull/117/commits/257e362ccd7918ba5e686428bb0f692fb57ce25d